config/config.toml.example

```diff
@@ -390,6 +384,16 @@ store-liveness-timeout = "5s"
 # If the size(in byte) of a transaction is large than `ttl-refreshed-txn-size`, it update the lock TTL during the 2PC.
 ttl-refreshed-txn-size = 33554432
 
+[tikv-client.async-commit]
+# Whether to enable the async commit feature. This feature reduces the latency of the two-phase commit.
+enable = false
+# The maximum allowed keys in a async commit transaction. Transactions with more keys than the limit
+# will be committed with normal 2PC way.
+keys-limit = 256
+# The maximum sum of length of secondary keys. Transactions will be committed with the normal 2PC way
```

Comment:
I'm afraid the users may not well understand what "secondary" means, and we will be asked frequently. How about simply limit the sum of all keys in the transaction?

Reply:
Makes sense. It's easier for users to understand and there is no big difference.